package.json

```diff
@@ -16,6 +16,7 @@
   },
   "devDependencies": {
     "@types/node": "^18.6.4",
+    "bedrock-protocol": "^3.25.0",
```

Comment:
should not be needed, mc-data or [mc-packets](https://github.com/PrismarineJS/minecraft-packets) should be used instead